php/libraries/Candidate.class.inc

```diff
@@ -504,12 +507,21 @@ class Candidate
     /**
      * Return the candidate's sex
      *
-     * @return string|null Male|Female|Other String describing the sex
-     *                     and null if scanner
+     * @return ?Sex
```

Comment:
```suggestion
     * @return Sex|null Sex value object
```

Reply:
I disagree. `?` for nullable is standard PHP, anyone who doesn't know it will have to learn it pretty quickly.

Reply:
sure. i don't really have a strong stance on this